packages/ses/test/error/assert-log.test.js

```diff
@@ -0,0 +1,337 @@
+// TODO The following two lines came from agoric-sdk with already supports ava.
```

Comment:
TODO

Reply:
Not sure what you want me to do here. In moving this package from the agoric-sdk monorepo to the SES-shim monorepo, I had to regress the testing to use tap or tape rather than AVA. I put this comment there to help when we reverse this regression, once SES-shim switches to use AVA.

I'll leave this unresolved, but I am not inclined to postpone merging till we resolve this.